flow/scripts/variables.yaml

```diff
@@ -169,6 +169,13 @@ SYNTH_MEMORY_MAX_BITS:
   default: 4096
   stages:
     - synth
+SYNTH_HDL_FRONTEND:
```

Comment:
@jeffng-or See here. I went with "HDL frontend" since we might start supporting VHDL through Verific or GHDL